src/pages/ContactUs/ContactUs.tsx

```diff
@@ -1,5 +1,92 @@
 import './ContactUs.scss';
+import hehehe from './photo/Group 124.svg';
```

Comment:
hehehe?) really?)

Reply:
I couldn't think of a suitable name at that moment)))))))))